src/api/index.spec.tsx

```diff
@@ -1,13 +1,16 @@
 /* global fetchMock */
+import configureStore from '../configureStore';
 import {
   actions as apiActions,
   initialState as defaultApiState,
 } from '../reducers/api';
-import configureStore from '../configureStore';
 
 import { HttpMethod, callApi, getVersion, logOutFromServer } from '.';
 
 describe(__filename, () => {
+  const defaultLang = process.env.REACT_APP_DEFAULT_API_LANG as string;
```

Comment:
why does that need `as string`?

Reply:
I think I needed it at one point, but I don't anymore. It was probably a red herring. I've removed it.